daemon/src/payout_curve.rs

```diff
@@ -101,8 +103,10 @@ fn calculate_payout_parameters(
             let right_bound = row[1] as u64;
             let long_amount_btc = row[2];
 
+            // TODO: cover the case where the maker pays funding fees
```

Comment:
❓ Are you going to do this as part of this PR? If not, I think we can safely skip this TODO.

Reply:
I'd prefer not to, as I will be working on this in the near future (like tomorrow). I will ensure that it's tracked in a separate issue.